### DISCUSSION TITLE
Option to exclude Blue Flag notifications from RCM window

### DISCUSSION THREAD

**Discussion by dylano** [f1multiviewer/issue-tracker#151] [Feature requests] [answered]
Blue flags are the least interesting messages from Race Control. On a track like Hungary they come fast & furious and dilute the usefulness of the RCM window. 

Suggest that they do not trigger the RCM notification sound and probably would hide them altogether by default, with perhaps a setting to show them in the stream (but I'd be unlikely to turn it on).

**Comment by Sloth-on-meth** [comment]
A general filter would be awesome. selecting ' i want to be notified if red flags, yellow flags etc happen'. like a menu with checkboxes

**Reply by f1multiviewer** [reply]
Feel free to create a discussion for this one, this is a bit more complicated, so I would love to see how many people would want this before diving into it :)

**Comment by f1multiviewer** [comment, marked as answer]
This is now done!
